src/cljs/nr/game_row.cljs

```diff
@@ -43,19 +43,64 @@
+     (tr [:lobby.watch "Watch"])]
+    (letfn [(join-fn [side]
+              #(do (join-game lobby-state game "watch" side)
+                   (resume-sound)))]
+      [:div.split-button
+       [:button {:on-click (join-fn nil)}
+        (tr [:lobby.watch "Watch"])]
+       [:button.dropdown-toggle {:data-toggle "dropdown"}
+        [:b.caret]]
+       [:ul.dropdown-menu.blue-shade
+        [:a.block-link {:on-click (join-fn "Corp")}
+       (tr [:lobby.corp-perspective "Corp Perspective"])]
+      [:a.block-link {:on-click (join-fn "Runner")}
+       (tr [:lobby.runner-perspective "Runner Perspective"])]
+      [:a.block-link {:on-click (join-fn nil)}
+       (tr [:lobby.both-perspective "Both"])]]])))
+
+(defn- watch-protected-game-button
+  [spectatorhands lobby-state game]
+  (if-not spectatorhands
+    [:button {:on-click #(if (:password game)
+                           (authenticated
+                             (fn [_]
+                               (swap! lobby-state assoc :password-game {:game game :action "watch"})))
+                           (do (join-game lobby-state game "watch")
+                               (resume-sound)))}
+     (tr [:lobby.watch "Watch"])]
+    (letfn [(join-fn
+              [side]
+              #(if (:password game)
+                 (authenticated
+                   (fn [_]
+                     (swap! lobby-state assoc :password-game {:game game :action "watch" :request-side side})))
+                 (do (join-game lobby-state game "watch" side)
+                     (resume-sound))))]
+      [:div.split-button
+       [:button {:on-click (join-fn nil)}
+        (tr [:lobby.watch "Watch"])]
+       [:button.dropdown-toggle {:data-toggle "dropdown"}
+        [:b.caret]]
+       [:ul.dropdown-menu.blue-shade
+        [:a.block-link {:on-click (join-fn "Corp")}
+       (tr [:lobby.corp-perspective "Corp Perspective"])]
+      [:a.block-link {:on-click (join-fn "Runner")}
+       (tr [:lobby.runner-perspective "Runner Perspective"])]
+      [:a.block-link {:on-click (join-fn nil)}
+       (tr [:lobby.both-perspective "Both"])]]])))
+
 (defn watch-button [lobby-state user game current-game editing]
   (when (can-watch? user game current-game editing)
     (if (not (:password game))
-      [:button {:on-click #(do (join-game lobby-state game "watch")
-                               (resume-sound))}
-       (tr [:lobby.watch "Watch"])]
-      [:button {:on-click #(if (:password game)
-                             (authenticated
-                               (fn [_]
-                                 (swap! lobby-state assoc :password-game {:game game :action "watch"})))
-                             (do (join-game lobby-state game "watch")
-                                 (resume-sound)))}
-       (tr [:lobby.watch "Watch"])])))
+      (watch-game-button (:spectatorhands game) lobby-state game)
+      (watch-protected-game-button (:spectatorhands game) lobby-state game))))
```

Comment:
```suggestion
      [watch-game-button (:spectatorhands game) lobby-state game]
      [watch-protected-game-button (:spectatorhands game) lobby-state game])))
```

Functions that return hiccup need to be wrapped in `[]` so reagent rerenders them correctly.